parm/config/config.base.emc.dyn

```diff
@@ -149,28 +149,35 @@ export DO_COUPLED="NO"
 export DO_WAVE="NO"
 export DO_OCN="NO"
 export DO_ICE="NO"
-export DO_AERO="NO"
+export DO_AERO=@DO_AERO@
```

Comment:
`DO_AERO` should be `NO` by default and be toggled as is done for the other components based on `APP`.